src/psyclone/tests/psyir/nodes/intrinsic_call_test.py

```diff
@@ -76,13 +76,11 @@ def test_intrinsiccall_intrinsic():
 
 
 def test_intrinsiccall_is_elemental():
-    '''Tests the is_elemental() method works as expected. There are
-    currently no elemental intrinsics so we can only test for
-    False.
-
-    '''
+    '''Tests the is_elemental() method works as expected.'''
```

Comment:
I think the changes to this test will need to be reverted.

Reply:
Reverted.